doc/src/vpr/debug_aids.rst

```diff
@@ -103,3 +103,42 @@ Available Operators
           * Ex. in_blocks_affected == 11 || temp_count == 9
   * +=
           * Ex. move_num += 8
+          
+          
+Setting moves maually during placement
+======================================
+
+Overview
+~~~~~~~~
+
+The manual move generator, when enabled, allows the user to specify the next move in placement. The user can set a breakpoint to stop the placer at a specific point and start making manual moves. After specifying the move, the user is presented with the costs of the move and can then decide to accept or reject it.
+
+How to use the manual move generator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, you should make sure the manual move option is enabled, as seen in the figure.
+
+.. image:: https://www.verilogtorouting.org/img/manualMoveCheckbox.png
+    :align: center
+    
+Then after pressing the “Proceed” button a window pops up that asks for the block you want to move, and the location you would want to move it to. 
+
+.. image:: https://www.verilogtorouting.org/img/manualMoveWindow.png
+    :align: center
+    
+By pressing the “calculate costs” button you can see all the costs of your move, as well as the placer’s move_outcome and then decide to reject or accept the move. 
+
+.. image:: https://www.verilogtorouting.org/img/moveCosts.png
+    :align: center
+
+If you want to make another manual move simply press proceed again and repeat the process, otherwise disable the manual move feature and continue.
+
+Selecting the block to move
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can choose the block you want to move by entering its block id or block name in the corresponding entry field, or you can simply click on that specific block. For better visualization, once you indicate your block through the entry field and press proceed, your block will be highlighted on the GUI.
+
+Selecting the to location
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The “to location” of your move consists of the x location, y location, and subtile. If you’re unsure of the subtile value input 0.
```

Comment:
(add at end)  as most (x,y) locations in most devices have only a single subtile.

Reply:
resolved